kolibri/plugins/learn/assets/src/vue/explore-page/index.vue

```diff
@@ -58,7 +81,103 @@
 
   @require '~core-theme.styl'
 
-  .topic-page
-    padding-left: 10px
+  .section-title
+    margin-top: 5vh
+    font-size: 1.2em
+    font-weight: 700
+
+  .temp-nav
+    width: 80px
+    height: 100%
+    position: fixed
+    top: 0
+    background-color: $core-bg-light
+
+  .learn-tool-bar
+    height: 30px
+    width: (200*3) + 24*3
```

Comment:
Looks like there are a few "special" numbers re-used in different places - e.g. `24` and `200`. would be cleaner to specify these as variables, e.g.

    $card-margin = 24